contracts/Comet.sol

```diff
@@ -1181,6 +1181,9 @@ contract Comet is CometMainInterface {
 
         int104 newPrincipal = principalValue(newBalance);
         updateBasePrincipal(account, accountUser, newPrincipal);
+        if (newPrincipal > 0) {
+            emit Transfer(address(0), account, presentValueSupply(baseSupplyIndex, unsigned104(newPrincipal)));
+        }
```

Comment:
Looks good 💪

Reply:
Should we move it lower down?

Reply:
Sure, we could. Should we put it at the very end, after `emit AbsorbDebt`?

I put it here because the transfer is actually happening in `updateBasePrincipal`.

Reply:
Done